app/controllers/contests_controller.rb

```diff
@@ -0,0 +1,18 @@
+class ContestsController < ApplicationController
+
+  def show
+    @contest = Contest.find_by(year: params[:year])
+    if @contest
+      render :show
+    else
+      render json: @contest.errors.full_messages, status: 404
+    end
+  end
+
+  def index
+
+  end
```

Comment:
I'd comment this out or finish the method instead of leaving it as a placeholder.

Reply:
Yes, sir!!